front_end/src/components/searchResultOverview/searchResultOverview.js

```diff
@@ -10,13 +10,49 @@ import {
   TablePagination
 } from "@material-ui/core";
 import SearchWidget from "../landing_page/search_widget/SearchWidget";
+import { connect } from "react-redux";
+import { displayResultsOverview } from "../../actions/searchResultActions";
 
 class searchResultOverview extends Component {
   render() {
```

Comment:
3
You can then access your CSS by decomposing 'classes' from props.
let { classes } = this.props;

Reply:
done